packages/bruno-js/src/bru.js

```diff
@@ -153,6 +155,63 @@ class Bru {
   sleep(ms) {
     return new Promise((resolve) => setTimeout(resolve, ms));
   }
+
+  sendRequest(requestConfig, callback) {
```

Comment:
Would it be possible to change send the request and pass a specific variable to be used in either the `pre` or `post` scripts?

For instance, I'd like to call `bru.sendRequest('./items/delete-item', { itemId: myCustomItemId  })`. This would allow developers to customize further their scripts by replacing "request-scoped" variables

Reply:
@thiagominiBruno provides an API, `bruno.runRequest('items/delete-item')`, to trigger another request already defined in Bruno.

The `sendRequest()` API, on the other hand, is used for making direct HTTP requests.

It seems logical to support passing variables when using `sendRequest()`. Could you please create a new GitHub issue outlining this requirement?

Reply:
Oh, I see @helloanoop , thanks for the info! So, my idea is actually related to passing variables using the `bruno.runRequest`, do we have that? If not, I'll create an issue asking for that feature

Reply:
@thiagomini Currently `bruno.runRequest()` does not support passing variables. Please create an issue for the same